spec/factories/cards.rb

```diff
@@ -2,6 +2,7 @@
   factory :card do
     original_text 'дом'
     translated_text 'house'
-    user
+    user_id ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.